context.go

```diff
@@ -72,6 +73,22 @@ func GetHeader(ctx Context) (abci.Header, bool) {
 	return val, ok
 }
 
+// WithCommitInfo sets the info on who signed the block in this Context.
+// Panics if already set.
+func WithCommitInfo(ctx Context, info CommitInfo) Context {
+	if _, ok := GetCommitInfo(ctx); ok {
+		panic("CommitInfo already set")
+	}
+	return context.WithValue(ctx, contextCommitInfo, info)
+}
+
+// GetCommitInfo returns the info on validators that signed
+// this block. Returns false if not present.
+func GetCommitInfo(ctx Context) (CommitInfo, bool) {
```

Comment:
It might be better to call it `MustCommitInfo`. There was [similar issue](https://github.com/iov-one/weave/issues/582) with the `BlockTime` function that was renamed to `MustBlockTime`.

Reply:
sounds good to me.

Reply:
yeah, but I can't see this anywhere by the way, so leaving this as is for consistency. If we are about to change things up to `Must` we shall do it in one go I think.